data/yaml/mspass.yaml

```diff
@@ -554,3 +555,43 @@ Metadata:
         type: double
         concept: Body wave magnitude
         readonly: true
+      arrival_time:
+        type: double
+        concept: Generic arrival time value.  Can be predicted value or measured.
+        readonly: false
+      Ptime_picked:
+        type: double
+        concept:  A measured P wave arrival time.
+        readonly: false
+      Stime_picked:
+        type: double
+        concept:  A measured S wave arrival time.
+        readonly: false
+      earth_model:
+        type: string
+        concept:  Name tag of an Earth model used to compute predicted phase data (time or slowness)
+        readonly: false
+      Ptime_modeled:
+        type: double
+        concept:  A P arrival time predicted from an Earth model.
+        readonly: false
+      Stime_modeled:
+        type: double
+        concept: An S phase arrival time predicted from an Earth model.
+        readonly: false
+      ux:
+        type: double
+        concept:  x-component (+east) of a measured slowness vector (s/km)
+        readonly: false
+      uy:
+        type: double
+        concept: y-component (+north) of a measured slowness vector (s/km)
+        readonly: false
+      ux_modeled:
+        type: double
+        concept: x-component (+east) of a slowness vector (s/km) computed from an Earth model.
+        readonly: false
+      uy_modeled:
+        type: double
+        concept: y-component (+north) of a slowness vector (s/km) computed from an Earth model.
+        readonly: false
```

Comment:
These belongs to a different yaml file. Also, in that file, we need to define corresponding keys in the Database schema first. Then, we can add them to the Metadata schema.